lms/templates/admin/macros.jinja

```diff
@@ -63,57 +63,51 @@
     {{ form_text_field(request, "Token URL", "token_url", lti_registration.token_url) }}
 {% endmacro %}
 
-{% macro instances_table(request, instances) %}
-<table class="table is-fullwidth">
-  <thead>
-    <tr>
-      <th>ID</th>
-      <th>Consumer Key</th>
-      <th>GUID</th>
-      <th>Issuer</th>
-      <th>Deployment ID</th>
-      <th></th>
-    </tr>
-  </thead>
-  <tbody>
-{% for instance in instances %}
-    <tr>
-      <td>{{instance.id}}</td>
-      <td>{{instance.consumer_key}}</th>
-      <td>{{instance.tool_consumer_instance_guid}}</td>
-      <td>{{instance.lti_registration.issuer}}</td>
-      <td>{{instance.deployment_id}}</td>
-      <td>
-        <a class="button" href="{{ request.route_url("admin.instance.id", id_=instance.id) }}">View</a>
-      </td>
-    </tr>
-{% endfor %}
+
+{% macro created_updated_fields(model) %}
+    {{ disabled_text_field("Created", model.created) }}
+    {% if model.updated %}
+        {{ disabled_text_field("Updated", model.updated) }}
+    {% endif %}
+{% endmacro %}
+
+{% macro object_list_table(request, route, objects, headers, fields) %}
+<div class="container">
+    <div class="table-container">
```

Comment:
This new `div` allow for the table to overflow

https://bulma.io/documentation/elements/table/#table-container
https://github.com/hypothesis/lms/issues/4410

Reply:
As mentioned elsewhere, I don't think this is working because the container isn't flexible.